src/core/or/policies.c

```diff
@@ -990,6 +991,60 @@ fascist_firewall_choose_address_rs(const routerstatus_t *rs,
   }
 }
 
+/** Like fascist_firewall_choose_address_base(), but takes in a smartlist
+ * <b>lspecs</b> consisting of a link specifier.
```

Comment:
There can be more than one link specifier in the list, so say: "one or more link specifiers".